cms/djangoapps/modulestore_migrator/tasks.py

```diff
@@ -1031,20 +1067,28 @@ def _migrate_node(
                     title=title,
                 )
             )
-            if container_type is None and target_entity_version is None and reason is not None:
-                # Currently, components with children are not supported
+
+            context.add_block_to_summary(container_type, is_unsupported=target_entity_version is None)
+            if container_type is None and target_entity_version is None:
+                # Currently, components with children are not supported, but they appear as unsupported in the summary.
                 children_length = len(source_node.getchildren())
-                if children_length:
-                    reason += (
-                        ngettext(
-                            ' It has {count} children block.',
-                            ' It has {count} children blocks.',
-                            children_length,
-                        )
-                    ).format(count=children_length)
+                for _ in range(children_length):
+                    context.add_block_to_summary(None, is_unsupported=True)
```

Comment:
I feel like it is not required to show children blocks in the summary separately as we are already adding the number of children blocks in the unsupported_reason field of parent.

Reply:
Removed in: https://github.com/openedx/edx-platform/pull/37653/commits/dba9a97e2917e75c3bec91a416864a11634385dd